### ISSUE TITLE
Implement and test torch.based resizing in preprocessing.

### ISSUE THREAD

**Issue by Sllambias** [Sllambias/yucca#138]
Based on the code in https://pytorch.org/docs/stable/_modules/torch/nn/functional.html#interpolate the interpolate uses other torch functions such as torch.nn.upsample_bicubic based on "modes" of interpolation.

Going straight to the underlying functions may allow using e.g. bicubic for 5D tensors (b,c,h,w,d) even though F.interpolate is limited to trilinear for 5D tensors. 
Then two separate resize functions could be made using e.g. trilinear for labels and bicubic for images.
See https://pytorch.org/docs/stable/generated/torch.nn.Upsample.html to investigate if this is possible.
Or find the underlying torch._C._nn.upsample_bicubic2d function (or its 3D equivalent if exists)
